### DISCUSSION TITLE
Sugestão: mais tempo card alerta

### DISCUSSION THREAD

**Discussion by gpuentes** [langflow-ai/langflow#2674] [Ideas]
Olá a todos que lerem essa solicitação. 
Quando estou no CANVA, olhando para os blocos de componentes e interajo com o icone de play. 
Um alerta do sistema indica visualmente se for `erro` ou `sucesso` , do lado esquerdo inferior .

**O Tempo do card com a mensagem deveria/poderia ser longa e uma opção de PAUSE quando mouse hover .**

No meu caso traduzindo para meu idioma:

Componente de criação de erros
Erro ao criar a saída de bate-papo componente: store_message não encontrado no ChatOutput

**Comment by gpuentes** [comment]
Hello to everyone who reads this request.
When I'm in CANVA, I look at the component blocks and interact with the play icon.
A system alert visually indicates if it is an error or success on the lower left side.

**The time of the card with the message should/could be long and an option to PAUSE when mouse hover.**

In my case translating into my language:

Error Creation Component
Error creating component chat output: store_message not found in ChatOutput

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
